src/main/docs/guide/installation/installHomebrew.adoc

```diff
@@ -9,6 +9,13 @@ You can find more information about Homebrew usage on their https://brew.sh/[hom
 
 You should now be able to run the Micronaut CLI.
 
+TIP: On macOS you may get a warning message saying that `“mn” cannot be opened because the developer cannot be verified.`.
+To resolve it, click the Apple menu, and select *System Settings*. Then browse to *Privacy & Security*, and scroll down
+to the *Security* section. There should be a warning that `"mn" was blocked`, with an *Allow Anyway* button.
+Clicking this button will fix the issue.
+
+See the [Homebrew documentation](https://docs.brew.sh/FAQ#why-cant-i-open-a-mac-app-from-an-unidentified-developer) for more details about this.
```

Comment:
Sorry, I used Cmd+K to add the link, but I keep forgetting that it adds Markdown-style links

```suggestion
See the https://docs.brew.sh/FAQ#why-cant-i-open-a-mac-app-from-an-unidentified-developer[Homebrew documentation] for more details about this.
```

Reply:
Thanks! I thought about adding a link there or to the github issue, but wasn't sure about it. This works.